game.py

```diff
@@ -20,7 +23,30 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+ 
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    wrong_guesses_count = 0
+
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+            print("Congratulations, you win!")
+            return
+        
+        guess = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if guess in snowman_word:
```

Comment:
Another way that this could be done is by checking the dictionary that was constructed with `build_letter_status_dict`. There are some advantages here that we've not talked about yet but it doesn't hurt to think through how it could be done!